CONTRIBUTING.md

```diff
@@ -552,113 +544,12 @@ example-alert-condition2.yml
-| install.mode        | yes       |         | The type of installation. Options are one of `link`, `nerdlet`, or `targetedInstall`                                                                                                                                                      |
-| install.destination | yes       |         | The destination of the installation. Based on the chosen install `mode`, options are `recipeName` for the `targetedInstall` mode, `url` for the `link` mode, or `nerdletId`, `nerdletState`, and `requiresAccount` for the `nerdlet` mode |
-| fallback            | no        |         | Uses the same fields as `install`                                                                                                                                                                                                         |
-
-#### Install fields
-
-- `targetedInstall`
-
-  - Uses the New Relic CLI to install instrumentation
-  - Example:
-
-  ```yaml
-  install:
-    mode: targetedInstall
-    destination:
-      recipeName: fake-install-recipe
-  ```
-
-- `nerdlet`
-
-  - Directs the user to a nerdlet to finish installing instrumentation
-  - Example:
-
-  ```yaml
-  install:
-    mode: nerdlet
-    destination:
-      nerdletId: some-nerdlet.id
-      nerdletState:
-        optionalKey: optional-value
-      requiresAccount: true
-  ```
-
-- `link`
-  - Directs the user to a documentation link to finish installing instrumentation
-  - Example:
-  ```yaml
-  install:
-    mode: link
-    destination:
-      url: https://docs.newrelic.com
-  ```
-
 ### Data sources
 
 Data sources represent a _single_ type of instrumentation, such as an agent, attributes on a transaction, a cloud provider integration, a third-party integration, etc.
-They can be broken out into two categories, CORE and COMMUNITY. The CORE data sources are provided by New Relic One and do _not_ exist within this repository, the COMMUNITY data sources _are_ defined within this repository.
+They can be broken out into two categories, CORE and COMMUNITY. The CORE data sources are provided by New Relic One and do _not_ exist within this repository, the COMMUNITY data sources _are_ defined within this repository. 
+
+To see the available CORE data sources, you can use NerdGraph for to explore the available data source ids and their corresponding metadata using [this query](https://api.newrelic.com/graphiql?#query=%7B%0A%20%20actor%20%7B%0A%20%20%20%20nr1Catalog%20%7B%0A%20%20%20%20%20%20search%28filter%3A%20%7Btypes%3A%20DATA_SOURCE%7D%29%20%7B%0A%20%20%20%20%20%20%20%20results%20%7B%0A%20%20%20%20%20%20%20%20%20%20...%20on%20Nr1CatalogDataSource%20%7B%0A%20%20%20%20%20%20%20%20%20%20%20%20id%0A%20%20%20%20%20%20%20%20%20%20%7D%0A%20%20%20%20%20%20%20%20%7D%0A%20%20%20%20%20%20%7D%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D%0A).
```

Comment:
Could we make this callout again in the Quickstart `dataSourceIds` section or link to this?

Reply:
heck yeah